extras/signers.json

```diff
@@ -45,13 +45,15 @@
     "davgarai": "0xAc1aA53108712d7f38093A67d380aD54B562a650"
   },
   "ozDefender": {
-    "checkpointer": "0xe9735f7D85a57BFb860c1E2c1c7B4F587BA0F6e7"
+    "checkpointer": "0xe9735f7D85a57BFb860c1E2c1c7B4F587BA0F6e7",
+    "checkpointer_v2": "0xfb7d0D001BC8D0bC998071C762BfF53EE31b725F"
   },
   "maxiPay": {
     "factory": "0x7bbac709a9535464690a435ca7361256496f13ce"
   },
   "keepers": {
     "oz_defender_checkpointer": "0xe9735f7D85a57BFb860c1E2c1c7B4F587BA0F6e7",
+    "oz_v2_defender_checkpointer" : "0xfb7d0D001BC8D0bC998071C762BfF53EE31b725F",
```

Comment:
I wonder if we need this in 2 places, or if it makes sense to just put all the keepers here?